jpo-ode-plugins/src/test/java/us/dot/its/jpo/ode/plugin/j2735/builders/BsmBuilderTest.java

```diff
@@ -15,36 +15,32 @@
eOffset></PathHistoryPoint><PathHistoryPoint><latOffset>129</latOffset><lonOffset>-634</lonOffset><elevationOffset>-75</elevationOffset><timeOffset>10170</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>173</latOffset><lonOffset>-588</lonOffset><elevationOffset>-62</elevationOffset><timeOffset>11560</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>77</latOffset><lonOffset>-645</lonOffset><elevationOffset>-66</elevationOffset><timeOffset>12570</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>29</latOffset><lonOffset>-500</lonOffset><elevationOffset>-42</elevationOffset><timeOffset>13750</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-45</latOffset><lonOffset>-123</lonOffset><elevationOffset>-43</elevationOffset><timeOffset>15710</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-103</latOffset><lonOffset>-20</lonOffset><elevationOffset>-43</elevationOffset><timeOffset>16680</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-54</latOffset><lonOffset>198</lonOffset><elevationOffset>-36</elevationOffset><timeOffset>19130</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-193</latOffset><lonOffset>-294</lonOffset><elevationOffset>19</elevationOffset><timeOffset>22209</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-382</latOffset><lonOffset>-467</lonOffset><elevationOffset>28</elevationOffset><timeOffset>23300</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-390</latOffset><lonOffset>-364</lonOffset><elevationOffset>18</elevationOffset><timeOffset>24320</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-178</latOffset><lonOffset>-120</lonOffset><elevationOffset>-3</elevationOffset><timeOffset>26240</timeOffset></PathHistoryPoint></crumbData></pathHistory><pathPrediction><radiusOfCurve>32767</radiusOfCurve><confidence>0</confidence></pathPrediction></VehicleSafetyExtensions></partII-Value></PartIIcontent><PartIIcontent><partII-Id>1</partII-Id><partII-Value><SpecialVehicleExtensions><doNotUse><doNotUse>0</doNotUse><connection><pivotOffset>100</pivotOffset><pivotAngle>28800</pivotAngle><pivots><true/></pivots></connection><units><TrailerUnitDescription><isDolly><false/></isDolly><width>200</width><length>600</length><height>40</height><mass>128</mass><frontPivot><pivotOffset>100</pivotOffset><pivotAngle>0</pivotAngle><pivots><false/></pivots></frontPivot><positionOffset><x>0</x><y>0</y></positionOffset></TrailerUnitDescription></units></doNotUse></SpecialVehicleExtensions></partII-Value></PartIIcontent><PartIIcontent><partII-Id>2</partII-Id><partII-Value><SupplementalVehicleExtensions><classDetails><keyType>0</keyType><role><basicVehicle/></role><hpmsType><none/></hpmsType><fuelType>0</fuelType></classDetails><vehicleData><height>38</height></vehicleData><doNotUse2><airTemp>191</airTemp></doNotUse2></SupplementalVehicleExtensions></partII-Value></PartIIcontent></partII></BasicSafetyMessage></value></MessageFrame></data></payload></OdeAsn1Data>");
```

Comment:
We should separate the XML in this file into individual XML files and make sure they include all possible optional values from the schema.

Reply:
Addressed